src/issue.rs

```diff
@@ -208,6 +209,28 @@ impl Issue {
     }
 }
 
+/// Returns `Severity` from `IssueSeverity`
+///
+/// Example:
+///
+/// ```rust
+/// use codespan_reporting::diagnostic::Severity;
+/// use ara_reporting::issue::IssueSeverity;
+///
+/// assert_eq!(Severity::Error, IssueSeverity::Error.into());
+/// ```
+impl From<IssueSeverity> for Severity {
```

Comment:
```suggestion
#[docs(hidden)]
impl From<IssueSeverity> for Severity {
```

Reply:
we don't want this to show up in the documentation ( https://docs.rs/ara_reporting/latest/ara_reporting/ )